linter.py

```diff
@@ -17,6 +17,30 @@ class Pyflakes(PythonLinter):
     defaults = {
         'selector': 'source.python'
     }
+    warning_msg_regex = r'''(?x)
```

Comment:
```suggestion
    warning_msg_regex = r'''(?x)
    warning_msg_regex = re.compile(r'''(?x)
```

We want to compile exactly *once* because the regex is static.  Doing so in `__init__` would still compile per linter run.  You can do this either here as a class member or just in the global scope.   Anyway, after that overriding `__init__` should not be needed anymore.